### PR TITLE
Add fetcher as a default feature

### PR DESCRIPTION
**Pull request by leshow**
I killed the progressbar and removed `indicatif`

`reqwest`, `zip`, and `directories` are optional, but default. so @lukaslueg you should be fine to run w/o bringing in any tokio deps by using:

```
[dependencies.headless_chrome]
default-features = false
``` 

Come to think of it, I need to add that to the readme

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -43,4 +42,5 @@ name = "headless_chrome"
 path = "src/lib.rs"
 
 [features]
+default = [ "reqwest", "directories", "zip" ]
```
Comment (lukaslueg) [inline Cargo.toml]: Could we make this a named feature in itself instead of dispersed in `default`? Something like `fetch`

Comment (leshow) [reply Cargo.toml]: yeah of course, good idea.

**Comment by leshow** [comment]
Seems like the CI is erroring on trying to run some non-rust code from the readme file. Anyone know how to prevent that?

File: README.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ If you get errors related to timeouts, you likely need to enable sandboxing eith
 
 By default, `headless_chrome` will download a compatible version of chrome to `XDG_DATA_HOME` (or equivalent on Windows/Mac). This behaviour can be optionally turned off, and you can use the system version of chrome (assuming you have chrome installed) by disabling the default feature in your `Cargo.toml`:
 
-```
```
Comment (lukaslueg) [inline README.md]: The `toml` is required because CI will pull in the README as an external doc and make sure that all code examples actually compile

**Comment by atroche** [comment]
LGTM!

**Pull request merged**